### PR TITLE
remove google analytics

### LINKED ISSUE #240: Remove Google Analytics
No longer using since we switch to PostHog

### PR DESCRIPTION
**Pull request by nathancovey**
## The Pull Request is ready

- [x] all github actions are passing
- [x] are changes backwards compatible?
- [x] fixes #240 
- [x] the branch follows the naming schema `issue-123-enable-x-does-not-disable-y`
- [x] the pull request has a sensible title

## Intention

We are no longer using Google Analytics, and I see no reason to use it again in the future.

## Review Points

Are there any other lines of code that need to be deleted?

## The code follows best practices

- [x] the code is readable
- [x] issues for follow-up tasks have been created
- [x] there is no `any` type used

### CODE REVIEW

File: packages/app/src/routes/AppRoutes.tsx [hunk 1]
```diff
@@ -10,12 +9,6 @@ import { ExtensionsPage } from '../components/Extensions/ExtensionsPage'
 import extensionsService from '../services/extensions.service'
 import { ExtensionsLayout } from '../layouts/ExtensionsLayout'
 
-const AppRoutesPageSubscription = () => {
```
Comment (rphovley) [inline packages/app/src/routes/AppRoutes.tsx:13]: Looks like this is being used a few lines later on line 16

**Review by rphovley — CHANGES_REQUESTED** [review]

File: packages/app/src/routes/AppRoutes.tsx (hunk 1 above)
Comment (nathancovey) [reply packages/app/src/routes/AppRoutes.tsx:13]: Good find

Comment (nathancovey) [reply packages/app/src/routes/AppRoutes.tsx:13]: Do I need to do anything else to approve your changes?

**Review by rphovley — APPROVED** [review]

**Comment by rphovley** [comment]
Looks good!

**Review by rphovley — APPROVED** [review]

**Pull request merged**